src/modules/systemlib/circuit_breaker_params.c

```diff
@@ -106,7 +105,7 @@ PX4_PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK);
  * @max 121212
  * @group Circuit Breaker
  */
-PX4_PARAM_DEFINE_INT32(CBRK_FLIGHTTERM);
+PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
```

Comment:
Actually going over the code again I am not sure if the default values for the circuit breakers are used correctly by the ros code. All other parameters are initialized here via the constructor: https://github.com/PX4/Firmware/blob/master/src/platforms/px4_parameter.h#L92